ui/apps/platform/src/Components/ScannerV4IntegrationBanner.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Banner, Flex, FlexItem } from '@patternfly/react-core';
+import { InfoCircleIcon } from '@patternfly/react-icons';
+
+import useRestQuery from 'hooks/useRestQuery';
+import useMetadata from 'hooks/useMetadata';
+import { fetchImageIntegrations } from 'services/ImageIntegrationsService';
+import { getVersionedDocs } from 'utils/versioning';
+import { scannerV4ImageIntegrationType } from 'types/imageIntegration.proto';
+
+function ScannerV4IntegrationBanner() {
+    const { version } = useMetadata();
+    const { data, loading, error } = useRestQuery(fetchImageIntegrations);
```

Comment:
All of the other existing options (via Redux, via existing hooks) to obtain this data default to `[]` while the request is in flight. Since an empty array is the property we are looking for to determine whether or not to show the banner, this would result in a flicker if the user _did_ have a scanner v4 integration.